src/uniref/components/entry/SequenceSection.tsx

```diff
@@ -1,12 +1,16 @@
 import { Card, LongNumber, Sequence } from 'franklin-sites';
```

Comment:
The SequenceSection.tsx is the same for both UniParc and UniRef. We have them one each instead of a shared one because of the different data type?

Reply:
I created a shared component wrapping the call to Franklin's sequence component that generates the correct infolist and also handles the default rendering of sequence depending on sequence length and screen size.

Also, I've taken the opportunity of reviewing usage of the sequence component to also not render by default the sequence in search results if the length is too big